index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <script src="./index.js"> </script>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <link rel="stylesheet" href="./master.css">
+    <title> Pokemon </title>
+</head>
+<body>
+        <div id="themeSong">
+        <audio controls autoplay
+        preload="metadata">
+                <source src="/Users/dylanabasiurua/Music/iTunes/iTunes Media/Music/TV/Themes Collection/Pokemon.mp3" type="audio/mp3"> </source>
+              </audio>
+        </div>
+         
+    <div> 
+        <div id="heyPoke"> 
+            <p> Pokemon </p>
+            <p> Gotta catch 'em all! </p>
+        </div>
+
+
+        <div id="mainPoke"> 
+            <h1 class="mainHeading"> Pokemon Battle Simulator</h1>
+            
+            <p id="instruction"> Click "get pokemon!" to display your two fighters! Click "battle!" to have pokemon battle</p>
+                <div class="bttns"> 
+                    <button id="button1" style="vertical-align:middle"><span>get pokemon! </span> </button>
+                
+                    <button id="button2"> battle! </button>
+                </div>
+        </div>
+
+
+
+        <div class="data">
```

Comment:
I see that you didn't end up using this div at all, so a good idea would be to clean it up before committing.